src/test/java/org/broadinstitute/dsde/consent/ontology/resources/TranslateResourceTest.java

```diff
@@ -39,6 +39,18 @@ public void testDatasetTranslate() throws IOException {
         assertEquals(response.readEntity(String.class), mockService.translateDataset(""));
     }
 
+    @Test
+    public void testParagraphTranslate() throws IOException {
+        String paragraph = "This could be a disease or anything else.";
+        Response response = gRule.client().target("/translate/paragraph")
+                .queryParam("paragraph", paragraph)
+                .request(MediaType.APPLICATION_JSON_TYPE)
+                .accept(MediaType.TEXT_PLAIN_TYPE)
+                .get();
+        assertEquals(200, response.getStatus());
+        assertEquals(mockService.translateParagraph(paragraph), response.readEntity(String.class));
```

Comment:
Note that this is testing two classes, the `TranslateResource` API and the `Translate` class. If `Translate` was injectable then you could test these classes independently.

Reply:
I agree with Phil's suggestion here. We should test the service and the resource separately. There are some poor testing practices on display in this test class. Let's pull the service call out and put that in a service class test.